### PR TITLE
Add vietnamese translation

### PR DESCRIPTION
**Pull request by hoannc54**
Hello, 
I translated your readme into Vietnamese. 

Nice to be contributing to your repo

### CODE REVIEW

File: README_vi.md [hunk 1]
```diff
@@ -0,0 +1,1647 @@
...
+0a2e358 HEAD@{0}: reset: moving to HEAD~2
+0254ea7 HEAD@{1}: checkout: moving from 2.2 to master
+c10f740 HEAD@{2}: checkout: moving from master to 2.2
+```
+
+Các reflog ở trên cho thấy một checkout từ master đến nhánh 2.2 và trở lại. Từ đó, có một thiết lập cứng để một commit cũ hơn. Hoạt động mới nhất được thể hiện ở đầu được gắn nhãn `HEAD@{0}`.
+
+Nếu nó chỉ ra rằng bạn vô tình di chuyển trở lại, các reflog sẽ chứa commit master chỉ đến (0254ea7) trước khi bạn vô tình giảm 2 commit
+
+If it turns out that you accidentally moved back, the reflog will contain the commit master pointed to (0254ea7) before you accidentally dropped 2 commits.
```
Comment (RichardLitt) [inline README_vi.md]: Missed a line. :)

File: README_vi.md [hunk 2]
```diff
@@ -0,0 +1,1647 @@
...
+* [Getting solid at Git rebase vs. merge](https://medium.com/@porteneuve/getting-solid-at-git-rebase-vs-merge-4fa1a48c53aa)
+* [Git Commands and Best Practices Cheat Sheet](https://zeroturnaround.com/rebellabs/git-commands-and-best-practices-cheat-sheet) - A Git cheat sheet in a blog post with more explanations
+* [Git from the inside out](https://codewords.recurse.com/issues/two/git-from-the-inside-out) - A tutorial that dives into Git's internals
+* [git-workflow](https://github.com/asmeurer/git-workflow) - [Aaron Meurer](https://github.com/asmeurer)'s howto on using Git to contribute to open source repositories
+* [GitHub as a workflow](https://hugogiraudel.com/2015/08/13/github-as-a-workflow/) - An interesting take on using GitHub as a workflow, particularly with empty PRs
+* [Githug](https://github.com/Gazler/githug) - A game to learn more common Git workflows
+
+## Scripts and Tools
+
+* [firstaidgit.io](http://firstaidgit.io/) A searchable selection of the most frequently asked Git questions
```
Comment (RichardLitt) [inline README_vi.md]: Can you translate the comments?

File: README_vi.md [hunk 3]
```diff
@@ -0,0 +1,1647 @@
+# Flight rules cho Git
+
+🌍
+*[English](README.md) ∙ [Español](README_es.md)  ∙  [Русский](README_ru.md) ∙ [简体中文](README_zh-CN.md)∙ [한국어](README_kr.md)*
```
Comment (RichardLitt) [inline README_vi.md]: Can you add Vietnamese to the other READMEs, too?

File: README_vi.md (hunk 1 above)
Comment (hoannc54) [reply README_vi.md]: I forgot to delete this line. I fixed this by delete this line

File: README_vi.md (hunk 2 above)
Comment (hoannc54) [reply README_vi.md]: Ok. I translated. You chan recheck.

File: README_vi.md (hunk 3 above)
Comment (hoannc54) [reply README_vi.md]: Ok. I added to  all other readmes

**Comment by RichardLitt** [comment]
Merged in https://github.com/k88hudson/git-flight-rules/commit/20e4b2c10d5705ae3042fb4b7722162fdeac7239. Thank you so much!

**Pull request closed without merging**